uwp/Ribbon/BackStage.md

```diff
@@ -78,7 +78,7 @@ Ribbon_backstage.Items.Add(Exit_backstagebutton)
 
 ![BackStage_img1](BackStage_images/BackStage_img1.jpeg)
 
-# BackStage in UWP Ribbon (SfRibbon(Touch Ribbon))
+## BackStage in UWP Ribbon (SfRibbon(Touch Ribbon))
```

Comment:
You have used the same content in both section. It will reproduce "Duplicate" issue again. So, can you give relevant section name for each section.